SU2_CFD/src/drivers/CMultizoneDriver.cpp

```diff
@@ -147,6 +147,10 @@ void CMultizoneDriver::StartSolver() {
 
   driver_config->Set_StartTime(StartTime);
 
+  for (iZone = 0; iZone < nZone; iZone++) {
+    config_container[iZone]->SetDelta_UnstTimeND(config_container[ZONE_0]->GetDelta_UnstTimeND());
+  }
+
```

Comment:
This is ugly and if someone has got the time right now, I'd like to have your suggestions what the 'right' approach might be.

Reply:
It's improved now by taking the minimum of all reference times as the reference time for all zones (not requiring any hard-coded values).